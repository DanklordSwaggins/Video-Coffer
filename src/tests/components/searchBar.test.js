import React from 'react';
import ReactShallowRenderer from 'react-test-renderer/shallow';
import SearchBar from '../../components/searchBar';
import configureStore from '../..//store/store'
import { Provider } from "react-redux";
import { BrowserRouter as Router } from "react-router-dom";
import {shallow, mount} from 'enzyme';

it('should render home', ()=>{
    const store = configureStore();
    const wrapper = mount(
        <Provider store={store}>
            <Router>
                <SearchBar />
            </Router>
        </Provider>
    );
    expect(wrapper).toMatchSnapshot();
})