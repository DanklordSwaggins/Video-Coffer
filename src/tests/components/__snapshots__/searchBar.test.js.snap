// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render home 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <Router
      history={
        Object {
          "action": "POP",
          "block": [Function],
          "createHref": [Function],
          "go": [Function],
          "goBack": [Function],
          "goForward": [Function],
          "length": 1,
          "listen": [Function],
          "location": Object {
            "hash": "",
            "pathname": "/",
            "search": "",
            "state": undefined,
          },
          "push": [Function],
          "replace": [Function],
        }
      }
    >
      <SearchBar>
        <div>
          <InputGroup
            size="lg"
          >
            <div
              className="input-group input-group-lg"
            >
              <FormControl
                onChange={[Function]}
                placeholder="Inject queries here..."
              >
                <input
                  className="form-control"
                  onChange={[Function]}
                  placeholder="Inject queries here..."
                />
              </FormControl>
            </div>
          </InputGroup>
          <div
            className="flex-container"
          />
        </div>
      </SearchBar>
    </Router>
  </BrowserRouter>
</Provider>
`;
